templates/rsyslog.conf.erb

```diff
@@ -67,6 +67,14 @@ $Umask <%= scope.lookupvar('rsyslog::umask') %>
 $OmitLocalLogging on
 <% end -%>
 
+# Settings for imjournal
+<% if scope.lookupvar('rsyslog::im_journal_statefile') -%>
+$IMJournalStateFile <%=scope.lookupvar('rsyslog::im_journal_statefile') %>
+<% end %>
+$ImjournalIgnorePreviousMessages <%=scope.lookupvar('rsyslog::im_journal_ignore_previous_messages') %>
```

Comment:
This field, and the new ones added below, will come up blank on operating systems without journald (which rsyslog might not handle properly), unless they're wrapped with a `scope.lookupvar` (much like the other parameters).